src/main/io/osd.c

```diff
@@ -600,6 +599,21 @@ static bool osdDrawSingleElement(uint8_t item)
 
             break;
         }
+		
+    case OSD_MOTOR_DIAG:
+        if(areMotorsRunning()) {
+            //float normalizedOutput = motorOutputHigh - motorOutputLow;
```

Comment:
Please don't just comment out code - remove it if it's not needed, or else add a comment stating when it should be re-activated.